Source/Application/Constants/UserStorage.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UserStorage.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/15/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import Foundation
+
+enum UserStorage {
+
+    @UserDefault(key: .selectedRepos, defaultValue: [String]())
```

Comment:
please take a look at [AppStorage](https://www.hackingwithswift.com/quick-start/swiftui/what-is-the-appstorage-property-wrapper) also example in `GeneralScreenObservable`

Reply:
@blyscuit The storing notified repositories part is quite small, so I included the integration in this PR as well. Please help check 🙏